src/napari_threedee/manipulators/base_manipulator.py

```diff
@@ -115,16 +120,12 @@ def __init__(
             [0, 0, 1, 1],
         ]
 
-        # initialize the arrow lines. if they weren't defined by the super class,
-        # initialize them as empty.
-        if not hasattr(self, '_initial_translator_normals'):
-            self._initial_translator_normals = np.empty((0, 3))
+        # initialize the translators.
+        self._set_initial_translation_vectors()
```

Comment:
To get around the problem with initializing without a layer, I think we should call this and `self._set_initial_rotator_normals()` in the `set_layers() method.; That way only when the layer is set is the rotator initialized.

I suppose we will also have to remove the visual from the scenegraph when the layer is changed.

Reply:
This seems important for 0.0.1